### PR TITLE
bump sync plugin to v0.1.19

### PR DESCRIPTION
**Pull request by gabemontero**
for https://bugzilla.redhat.com/show_bug.cgi?id=1461235

@bparees @dongboyan77 FYI

### CODE REVIEW

**Comment by gabemontero** [comment]
[test]

**Comment by gabemontero** [comment]
@bparees I also made a very small tweak to the README ... I'll wait a bit to see if you have time to review before merging

File: README.md [hunk 1]
```diff
@@ -165,7 +165,7 @@ Visit [the upstream repository](https://github.com/openshift/jenkins-plugin), as
...
 
 * **OpenShift Sync Plugin**
-Visit [the upstream repository](https://github.com/openshift/jenkins-sync-plugin) as well as the [Jenkins plugin wiki](https://wiki.jenkins-ci.org/display/JENKINS/OpenShift+Sync+Plugin).  This plugin facilitates the integration between the OpenShift Jenkinsfile Build Strategy and Jenkins Pipelines.  See the [OpenShift documentation](https://docs.openshift.com) for more details. 
+Visit [the upstream repository](https://github.com/openshift/jenkins-sync-plugin) as well as the [Jenkins plugin wiki](https://wiki.jenkins-ci.org/display/JENKINS/OpenShift+Sync+Plugin).  This plugin facilitates the integration between the OpenShift Jenkinsfile Build Strategy and Jenkins Pipelines.  It also facilitates auto-configuration of the pod templates for the Kukbernetes Plugin.  See the [OpenShift documentation](https://docs.openshift.com) for more details.
```
Comment (bparees) [inline README.md]: typo in kubernetes.

and s/pod template/slave pod templates/

**Comment by gabemontero** [comment]
corrections pushed ... thanks @bparees

**Comment by bparees** [comment]
lgtm

**Comment by gabemontero** [comment]
[merge]

**Comment by bparees** [comment]
@stevekuznetsov uh.  i'm not sure how this merged so fast?  it didn't have passing tests for the most recent commit.

**Pull request merged**